components/groups.js

```diff
@@ -125,6 +125,62 @@ SteamCommunity.prototype.leaveGroup = function(gid, callback) {
 	});
 };
 
+SteamCommunity.prototype.getAllAnnouncements = function(gid, time, callback) {
+	if(typeof gid === 'string') {
+		gid = new SteamID(gid);
+	}
+
+	if(typeof time === 'function') {
+		callback = time;
+		time = new Date(0); // The beginnig of time...
+	}
+
+	var self = this;
+	this.request({ 
+		"uri": "https://steamcommunity.com/gid/" + gid.getSteamID64() + "/rss/" 
+	}, function(err, response, body) {
+		if(!callback) {
+			return;
+		}
+
+		if(err || response.statusCode >= 400) {
+			callback(err || new Error("HTTP error " + response.statusCode));
+			return;
+		}
+
+		if(self._checkCommunityError(body, callback)) {
+			return;
+		}
+
+		xml2js.parseString(body, function(err, results) {
+			if(err) {
+				return callback(err);
+			}
+
+			if(!results.rss.channel[0].item) {
+				return callback(null, []);
+			}
+
+			var announcements = results.rss.channel[0].item.map(function(announcement) {
+				var splitLink = announcement.link[0].split('/');
+				return {
+					headline: announcement.title[0],
+					content:  announcement.description[0],
+					date:     new Date(announcement.pubDate[0]),
+					author:   announcement.author[0], // Unfortunately, the RSS feed likes to give us personanames not steamid's
+					aid:      splitLink[splitLink.length - 1] // The ID after the last /
+					// Note this is marked as guid (gid?) in the rss feed but can also be obtained from link
```

Comment:
Long numbers in Steam that aren't SteamIDs are nearly always GIDs (GlobalIDs). They're global to the Steam system and contain a boxID (the ID of the server where they were generated), startup time of the server that generated it, and a sequential counter.